### PR TITLE
Oop intro

### PR DESCRIPTION
**Pull request by YuvalRi**
A few remarks:
1. The field 'model' can be represented by one of the three options in the exercise. A case where an input does not include none of these options did not define in the exercise instructions. Therefore, I chose by myself a message that will be printed in that kind of cases. 
2. I was skeptical about 'if else' appearing in the constructor function. I started to write the code with no 'if else' in it, but then I found myself repeat code lines in different functions. That is why I finally decided to include 'if else' in the constructor function. I hope that is not a big deal. 
3. All functions in the exercises are of 'snake_case' type, different from the function written in the exercise.
4. linter helps me to keep the code aesthetic.
5. I enjoyed writing the code.

### CODE REVIEW

**Comment by ofekshmu** [comment]
Before I start checking your PR i noticed a couple of things.
1. commit names: 'exercise_A', 'ex4_completed' and so on are BAD NAMES. Your commit should be informative, once you go back to a commit you want to have a general knowledge of what changed, You should always find the best description for your case.
2. I love your feedback at the top of the PR, I can see you understand what's going on, you have your own feedback and you are aware of the thing that might be out of the ordinary, and its important for you to let me know.

**Comment by ofekshmu** [comment]
* You commited all your recursion exercises onto this branch. Don't delete those now, but they will result in conflicts when you try to merge.
* You also commited the __pycache__ folder which should not be commited onto git.

**Comment by ofekshmu** [comment]
> A few remarks:
> 
> 1. The field 'model' can be represented by one of the three options in the exercise. A case where an input does not include none of these options did not define in the exercise instructions. Therefore, I chose by myself a message that will be printed in that kind of cases.
> 2. I was skeptical about 'if else' appearing in the constructor function. I started to write the code with no 'if else' in it, but then I found myself repeat code lines in different functions. That is why I finally decided to include 'if else' in the constructor function. I hope that is not a big deal.
> 3. All functions in the exercises are of 'snake_case' type, different from the function written in the exercise.
> 4. linter helps me to keep the code aesthetic.
> 5. I enjoyed writing the code.

1. Good thinking outside of the box.
2. It's perfectly ok. Great!
3.  Perfect. That is my bad and I will fix it.
4. exactly!
5. I'm happy you did!

File: oopIntro/chair_module.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        if self.ammunition == 0:
+            return True
+        else:
+            return False
+
+    def shoot(self):
+        '''
+        A function which cause the gun to shoot
+        '''
+        if (self.ammunition != 0):
```
Comment (ofekshmu) [inline oopIntro/chair_module.py:52]: Why the parentheses?

**Review by ofekshmu — CHANGES_REQUESTED** [review]
I almost had nothing to add about the assignment! You did a great job!

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+    "python.linting.pylintEnabled": false,
+    "python.linting.enabled": true,
+    "python.linting.pycodestyleEnabled": true,
+    "python.linting.pydocstyleEnabled": false
+}
```
[the remark this answers is not in the document]
Comment (YuvalRi) [reply .vscode/settings.json:6]: Added to .gitignore

File: oopIntro/chair_module.py [hunk 3]
```diff
@@ -0,0 +1,70 @@
+import time
+from ShowFunc import show
+
+# A
+
+
```
[the remark this answers is not in the document]
Comment (YuvalRi) [reply oopIntro/chair_module.py:6]: your'e wrong. The linter expects 2 blank lines between #A to class.

File: oopIntro/chair_module.py [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+        return self.model
+
+    def is_empty(self):
+        '''
+        Returns True if ammunition is empty and false otherwise
+        '''
+        if self.ammunition == 0:
+            return True
+        else:
+            return False
```
[the remark this answers is not in the document]
Comment (YuvalRi) [reply oopIntro/chair_module.py:46]: What do you mean by all other numbers? I know that 1 represents True statement and 0 represents False statement. The change you wrote added.

File: oopIntro/chair_module.py (hunk 1 above)
Comment (YuvalRi) [reply oopIntro/chair_module.py:52]: Changed. There are no parentheses in 'if else' statements at all

**Review by ofekshmu — APPROVED** [review]

**Pull request merged**